src/api/l_event.c

```diff
@@ -110,6 +111,12 @@ StringEntry lovrKeyboardKey[] = {
   { 0 }
 };
 
+StringEntry lovrPermission[] = {
+  [AUDIO_CAPTURE_PERMISSION] = ENTRY("audio_capture"),
```

Comment:
It is kind of silly, but the "lovely" enum style is to string multiple words together like `audiocapture`

Reply:
oh okay :sweat_smile: